docs/scarpet/api/Auxiliary.md

```diff
@@ -142,6 +143,31 @@ Available shapes:
      * `height` - height of the cyllinder, defaults to `0`, so flat disk. Can be negative.
      * `level` - level of details, see `'sphere'`.
 
+ * `'polygon'`:
+   * Required attributes:
+     * `points` - list of points defining vertices of the polygon
+   * Optional attributes:
+     * `relative` - list of bools. vertices of the polygon that affected by 'follow'. Could be a single bools to affact allpoints too. Default means that every point is affacted.
+     * `mode` - how those points are connected. may be "polygon"(default),"strip" or "triangles". "polygon" means that it will be viewed as vertices of a polygon center on the first one. "strip" means that it will be viewed as a triangles strip. "triangles" means that it will be viewed as some triangles that are not related to each other (therefor length of `points` in this mode have to be a multiple of 3).
+     * `inner` - if `true` it will make the inner edges be drawn as well. 
+     * `doublesided` - if `true` it will make the shapes visible from the back as well. Default is `true`. 
+
+ * `'block'` \ `'item'`:
+   * Required attributes:
+     * `pos` - position of the object.
+     * `item` - the object to show. could have NBT. is a item tuple or a string-id (exclusive to `'item'`)
+     * `block` - the object to show. could have NBT. is a block value or a string-id (exclusive to `'block'`)
+   * Optional attributes:
+     * `facing` - object's direction, where its facing. Possible options are: `player` (default, it
+     always rotates to face the player), `north`, `south`, `east`, `west`, `up`, `down`
+     * `tilt`, `lean`, `turn` - additional rotations along all three axis
+     * `scale` - scale of it in 3 axis-direction. should be a number or a list of 3 numbers (x,y,z).
+     * `skylight`, `blocklight` - light level. omit it to use local light level. should between 0~15.
+     * `variant` - one of "none","third_person_left_hand","third_person_right_hand","first_person_left_hand",
+     "first_person_right_hand","head","gui","ground","fixed". In addition to the literal meaning, it can also be used to use special models of tridents and telescopes. (exclusive to `'item'`)
```

Comment:
shoudn't 'none' be the default? it isn't

Reply:
ok